virt-v2v/entrypoint.go

```diff
@@ -150,14 +114,21 @@ func customizeVM(source string, xmlFilePath string) error {
 	return nil
 }
 
-func buildCommand() []string {
-	virtV2vArgs := []string{"-v", "-x"}
-	source := os.Getenv("V2V_source")
-
-	if !isValidSource(source) {
-		fmt.Printf("virt-v2v supports the following providers: {OVA, vSphere}. Provided: %s\n", source)
-		os.Exit(1)
+func convertVirtV2vInPlace() error {
+	args := []string{"-v", "-x", "-i", "libvirtxml"}
+	args = append(args, "--root")
+	if val, found := os.LookupEnv("V2V_RootDisk"); found {
+		args = append(args, val)
+	} else {
+		args = append(args, "first")
 	}
+	args = append(args, "/mnt/v2v/input.xml")
```

Comment:
I'm not sure if we have the domain available, you might need to specify all disks `virt-v2v-in-place -i disk IMG1 IMG2 ...`